messaging/src/main/java/org/axonframework/serialization/upcasting/event/IntermediateEventRepresentation.java

```diff
@@ -21,6 +21,7 @@
 import org.axonframework.serialization.LazyDeserializingObject;
 import org.axonframework.serialization.SerializedObject;
 import org.axonframework.serialization.SerializedType;
+import org.axonframework.serialization.Serializer;
```

Comment:
Given the changes which have occurred, we should remove this import statement too.